ui/pom.xml

```diff
@@ -38,16 +38,9 @@
         <configuration>
           <filesets>
             <fileset>
-              <directory>packages/models</directory>
+              <directory>.</directory>
               <includes>
-                <include>packages/models/dist/**</include>
-                <include>packages/models/node_modules/**</include>
-                <include>packages/ui/dist/**</include>
-                <include>packages/ui/node_modules/**</include>
-                <include>packages/services/dist/**</include>
-                <include>packages/services/node_modules/**</include>
-                <include>node/**</include>
-                <include>node_modules/**</include>
+                <include>**/dist/</include>
               </includes>
```

Comment:
I think we need here also:                
<include>**/node_modules/</include>
<include>**/node/</include>

Reply:
`mvn clean` is supposed to clean up / remove build artifacts. UI build artifacts are only in the `dist/` folders. So I don't agree here.